### PR TITLE
Feature/internal news labels

### PR DESCRIPTION
**Pull request by Gunvor4**
## Proposed changes

- Added labels to private events and articles
- Changed cropping of pictures in event list
- Changed color of 'private' button on admin page both for events and articles
- Added new translations

## Checklist

_(If any of the points are not relevant, mark them as checked)_

- [x] Created tests that fail without the changes, if relevant/possible
- [x] Made sure that your code conforms to [the code style guides](https://github.com/MAKENTNU/web/blob/main/CONTRIBUTING.md#code-style-guides) and that any [common code smells](https://github.com/MAKENTNU/web/blob/main/CONTRIBUTING.md#code-review-guideline-code-smells) have been addressed
  - (It's not intended that you read through this whole document, but that you get yourself an overview over its contents, and that you keep it in mind while taking a second look at your code before opening a pull request)
- [x] Added sufficient documentation - e.g. as docstrings or in the [README](https://github.com/MAKENTNU/web/blob/main/README.md), if suitable
- [x] Added your changes to the ["Unreleased" section of the changelog](https://github.com/MAKENTNU/web/blob/main/CHANGELOG.md#unreleased) - mainly the changes that are of particular interest to users and/or developers, if any
- [x] Added a "Deployment notes" section above, if anything out of the ordinary should be done when deploying these changes to the server

### CODE REVIEW

File: web/templates/web/index.html
Comment (ddabble) [inline web/templates/web/index.html:39]: See my comment on `event_list.html` about the ribbon classes variable names 🙂

**Review by ddabble — CHANGES_REQUESTED** [review]
Sorry for taking so long to review! Overall, this looks very good, though I have a few change requests: Apart from resolving the conflicts with `dev`, see my comments below 😊

Also, I see that you've checked the "Added your changes to the ["Unreleased" section of the changelog](https://github.com/MAKENTNU/web/blob/main/CHANGELOG.md#unreleased)" checkbox, which I interpret as you thinking that it's not necessary (correct me if I'm wrong)..? I would argue that it could be useful for e.g. users of the page to see this in the changelog, since it's a fairly visible change 🙂

Lastly, some of the commits' messages could benefit from adding just a small, concise explanation of _why_ the changes were done, but this is not necessary to amend 😛 Other than that, nice job! Particularly fond of the code formatting and your commit structure 😊

File: web/static/web/css/style.css
Comment (ddabble) [inline web/static/web/css/style.css:127]: Is it necessary to define a new, custom `enlarged` class, instead of using one of the classes Fomantic-UI has already defined? (see https://fomantic-ui.com/elements/label.html#size) 🤔

**Comment by ddabble** [comment]
Oh also - this is absolutely not required, but it would be really nice if you could add one or more screenshots of the (main) things that have been changed visually, to the PR description 😄 If you don't already know, either upload files by clicking the "button" at the bottom of the PR description text field (depicted below), or copy a (cropped) screenshot to your clipboard and just paste it into the description text field - which will automatically upload it 🙂

image

(I just remembered that doing this is generally good practice - especially to aid any people reviewing the PR - and so I added some screenshots to (the already merged) #636 and #637 - in case it wasn't clear exactly what I meant :))

File: locale/nb/LC_MESSAGES/django.po
Comment (Gunvor4) [reply locale/nb/LC_MESSAGES/django.po]: Ok. I didn't change this, since I already had quite a lot of trouble with the translations

File: news/templates/news/event/event_list.html
Comment (Gunvor4) [reply news/templates/news/event/event_list.html]: I now changed this

File: news/templates/news/event/event_list_contents.html
Comment (Gunvor4) [reply news/templates/news/event/event_list_contents.html]: Done 🙂

File: news/templates/news/news_base_detail.html
Comment (Gunvor4) [reply news/templates/news/news_base_detail.html]: Added an extra block, removed the if statement, and included the translate tag with the "event" context in event_detail.html, and the translate tag with the "article" context in article_detail.html

File: news/templates/news/news_base_detail.html
Comment (Gunvor4) [reply news/templates/news/news_base_detail.html]: The event class was added because while working with the internal news labels, I obviously made a change that caused the space between the picture and the admin panel button on the event detail page to disappear. But I have removed it, and instead added a margin-bottom to the news-obj-image class.

File: web/static/web/css/style.css
Comment (Gunvor4) [reply web/static/web/css/style.css]: Done 🙂

File: web/static/web/css/style.css
Comment (Gunvor4) [reply web/static/web/css/style.css]: I've now tried to combine them in a better way.

File: web/templates/web/index.html
Comment (Gunvor4) [reply web/templates/web/index.html]: I now changed this

File: web/static/web/css/style.css
Comment (Gunvor4) [reply web/static/web/css/style.css]: Probably not :-P I now removed the enlarged class, and instead use the Fomantic-UI classes :-) I wasn't aware of this at first.

**Comment by Gunvor4** [comment]
Articles
Events
News detail page

**Review by ddabble — CHANGES_REQUESTED** [review]
Veery nice 🤩 I'd say the code is now close to as high a quality as it can get, though I have just a few more requests before merging; see below 😊

File: CHANGELOG.md
Comment (Gunvor4) [reply CHANGELOG.md:15]: Done!

File: src/news/static/news/css/news_labels.css
Comment (Gunvor4) [reply src/news/static/news/css/news_labels.css:29]: I think this comment drowned in something else the first time :-P I've now added a new class news-label on both date and internal ribbon labels, which cleaned up the CSS code considerably :-)

File: src/news/templates/news/article/article_list_contents.html
Comment (Gunvor4) [reply src/news/templates/news/article/article_list_contents.html:1]: Done!

File: src/news/templates/news/event/event_list_contents.html
Comment (Gunvor4) [reply src/news/templates/news/event/event_list_contents.html:1]: ... and done!

**Review by ddabble — APPROVED** [review]
Excellent! 🥳

(I won't merge the PR myself, as it's somewhat customary to let a maintainer merge their own PRs (since they'll get credit for the merge commit) - even though this really doesn't matter 😅)

**Pull request merged**
